lib/src/utils/themes/code_block_theme.dart

```diff
@@ -0,0 +1,130 @@
+  final TextStyle regexp;
+  final TextStyle link;
+  final TextStyle title;
+  final TextStyle symbol;
+  final TextStyle bullet;
+  final TextStyle number;
+  final TextStyle section;
+  final TextStyle meta;
+  final TextStyle type;
+  // ignore: non_constant_identifier_names
+  final TextStyle built_in;
+  // ignore: non_constant_identifier_names
+  final TextStyle builtin_name;
+  final TextStyle params;
+  final TextStyle attr;
+  final TextStyle subst;
+  final TextStyle formula;
+  final TextStyle addition;
+  final TextStyle deletion;
+  // ignore: non_constant_identifier_names
+  final TextStyle selector_id;
+  // ignore: non_constant_identifier_names
+  final TextStyle selector_class;
+  final TextStyle doctag;
+  final TextStyle strong;
+  final TextStyle emphasis;
+
+  const CodeBlockTheme({
+    required this.root,
+    required this.comment,
+    required this.quote,
+    required this.tag,
+    required this.attribute,
+    required this.keyword,
+    required this.selector_tag,
+    required this.literal,
+    required this.name,
+    required this.variable,
+    required this.template_variable,
+    required this.code,
+    required this.string,
+    required this.meta_string,
+    required this.regexp,
+    required this.link,
+    required this.title,
+    required this.symbol,
+    required this.bullet,
+    required this.number,
+    required this.section,
+    required this.meta,
+    required this.type,
+    required this.built_in,
+    required this.builtin_name,
+    required this.params,
+    required this.attr,
+    required this.subst,
+    required this.formula,
+    required this.addition,
+    required this.deletion,
+    required this.selector_id,
+    required this.selector_class,
+    required this.doctag,
+    required this.strong,
+    required this.emphasis,
+  });
+
+  Map<String, TextStyle> _toMap() => {
+        'root': root,
+        'comment': comment,
+        'quote': quote,
+        'tag': tag,
+        'attribute': attribute,
+        'keyword': keyword,
+        'selector-tag': selector_tag,
+        'literal': literal,
+        'name': name,
+        'variable': variable,
+        'template-variable': template_variable,
+        'code': code,
+        'string': string,
+        'meta_string': meta_string,
+        'regexp': regexp,
+        'link': link,
+        'title': title,
+        'symbol': symbol,
+        'bullet': bullet,
+        'number': number,
+        'section': section,
+        'meta': meta,
+        'type': type,
+        'built_in': built_in,
+        'builtin-name': builtin_name,
+        'params': params,
+        'attr': attr,
+        'subst': subst,
+        'formula': formula,
+        'addition': addition,
+        'deletion': deletion,
+        'selector-id': selector_id,
+        'selector-class': selector_class,
+        'doctag': doctag,
+        'strong': strong,
+        'emphasis': emphasis,
+      };
+
+  String toKey(String key) => key.replaceAll('-', '_');
```

Comment:
This is for backward compatability. IIUC, the code block widget uses the `[]` operator with the dash (`-`) and expects a variable to be resolved. I haven't tested it, but this is something that you might want to watch out for.